### ISSUE TITLE
Swift SDK 0.23.1 doesn't compile on Linux and Travis is failing on master

### ISSUE THREAD

**Issue by csantanapr** [watson-developer-cloud/swift-sdk#773]
Trying to build a new image for OpenWhisk using latest Watson Swift SDK.

It doesn’t compile on Linux
I notice the repo has Travis CI enable, but it’s being ignore and things getting merge into master
I get the same error trying to build swift image for OpenWhisk.

```
/home/travis/build/watson-developer-cloud/swift-sdk/Source/VisualRecognitionV3/VisualRecognition+CoreML.swift:18:8: error: no such module 'CoreML'
import CoreML
```
What's the point on having Travis if it's currently being ignore and stuff getting merged into `master` in a bad state? Sorry but no sorry 😈 

https://travis-ci.org/watson-developer-cloud/swift-sdk/jobs/355689476

**Comment by csantanapr** [comment]
Same thing on latest build yesterday
https://travis-ci.org/watson-developer-cloud/swift-sdk/jobs/358710997

**Comment by csantanapr** [comment]
I guess I can hack with a workaround and ping an old version of the SDK until this get's fixed
```
dependencies: [
        .package(url: "https://github.com/watson-developer-cloud/swift-sdk", .exact("0.22.0"))
    ],
```

**Comment by germanattanasio** [comment]
Hi @csantanapr, 

Sorry for the troubles. We did a release for Think where we added support for CoreML and I think that library is not supported in Linux yet. 

We need to investigate how to maintain the compatibility with Linux.

**Comment by glennrfisher** [comment]
Thanks for pointing this out, @csantanapr! I published a fix with [v0.23.2][1].

Our team has a love-hate relationship with Travis. It's fantastic when it works, but their macOS support is not great (we've had builds remain queued for 24+ hours) and our tests sporadically timeout because of service issues.

We've gotten a little lax with Travis builds as a result. Thanks for calling us out on it—I'll try to see what I can do to fix some of the tests and get more consistent builds with Travis.

[1]: https://github.com/watson-developer-cloud/swift-sdk/releases/tag/v0.23.2
